tools/cockpit.spec

```diff
@@ -235,9 +253,10 @@ echo "m4_define(VERSION_NUMBER, %version)" > version.m4
 autoreconf -fvi -I tools
 #
 %configure \
-    %{?selinux_configure_arg} \
+    --disable-silent-rules \
```

Comment:
these 2 lines should remain as-is. the selinux one we need for selinux, and disable-silent-rules could be different patch that could be upstreamed.

Reply:
fixed